marlowe-dashboard-client/src/Pickup/View.purs

```diff
@@ -36,9 +36,9 @@ renderPickupState wallets newWalletNicknameKey pickupState =
 renderPickupCard :: forall p. Maybe Card -> WalletNicknameKey -> WalletLibrary -> HTML p Action
 renderPickupCard pickupCard newWalletNicknameKey wallets =
   div
-    [ classNames $ [ "absolute", "top-0", "bottom-0", "left-0", "right-0", "z-10", "flex", "flex-col", "justify-end", "md:justify-center", "bg-transgray" ] <> hideWhen (isNothing pickupCard) ]
+    [ classNames $ Css.cardWrapper $ isNothing pickupCard ]
     [ div
-        [ classNames $ [ "shadow-md", "bg-white", "mx-1", "md:mx-auto", "md:w-card" ] <> hideWhen (isNothing pickupCard) ]
+        [ classNames $ Css.card ]
```

Comment:
I think it should be `classes Css.cardClasses`

Reply:
I mean so that it makes sense when extending it `classes $ Css.cardClasses <> (map ClassName ["something"])`

Reply:
although maybe this persuades me to forget `ClassName` and just use `String`. Now I'm confused 🤔

Reply:
Css.card seems fine